sql/main/bookend.sql

```diff
@@ -0,0 +1,61 @@
+CREATE OR REPLACE FUNCTION first_sfunc(internal, anyelement, "any")
```

Comment:
Should we put these "internal" functions in an "internal" schema? I guess one would never call these functions directly, so maybe move out of PUBLIC?